File: Desafio de projetos/Classificador de Nível de Herói.js
quatXP = '';
nomeHeroi = '';

switch (quatXp){
    case 'Se XP for menor do que 1.000':
        console.log('Ferro')
        break

    case 'Se XP for menor do que 1.001 e 2.000':
        console.log('Broze')
        break
    
        case 'Se XP for entre 2.001 e 5.000':
            console.log('Prata')
            break        
        
        case 'Se XP for entre 5.001 e 7.000 ':
                console.log('Ouro')
                break

        case 'Se XP for entre 7.001 e 8.000':
            console.log('Platina')
                break
        case 'Se XP for entre 8.001 e 9.000':
                console.log('Ascendente')
                break

        case 'Se XP for entre 9.001 e 10.000':
                console.log('Imortal')
                break

        case 'Se XP for maior ou igual a 10.001':
            console.log('Radiante')
                break        
}

console.log('O Herói {nomeHeroi} está no nível de {quatXP }');

File: Desafio de projetos/Classificador de Nível de Herói.py
nome_heroi = input("Digite o nome do heroi: ")
xp = int(input("Digite a quantidade de XP do seu Heroi: "))

# Determinando o nível com base no XP
if xp < 1000:
    nivel = "Ferro"
elif 1001 <= xp <= 2000:
    nivel = "Bronze"
elif 2001 <= xp <= 5000:
    nivel = "Prata"
elif 5001 <= xp <= 7000:
    nivel = "Ouro"
elif 7001 <= xp <= 8000:
    nivel = "Platina"
elif 8001 <= xp <= 9000:
    nivel = "Ascendente"
elif 9001 <= xp <= 10000:
    nivel = "Imortal"
elif xp >= 10001:
    nivel = "Radiante"

print(f"O Herói {nome_heroi} está no nível de {nivel}")